src/igvfd/types/source.py

```diff
@@ -0,0 +1,24 @@
+from snovault import (
+    collection,
+    load_schema,
+    calculated_property
+)
+from .base import (
+    Item,
+    paths_filtered_by_status,
+    ALLOW_CURRENT,
+    DELETED,
+)
+
+
+@collection(
+    name='sources',
+    unique_key='source:name',
+    properties={
+        'title': 'Sources',
+        'description': 'Listing of sources',
```

Comment:
Can we change this to Listing of external sources so that it is clear that internal IGVF labs don't go in source